mesonbuild/backend/ninjabackend.py

```diff
@@ -1185,7 +1185,7 @@ def generate_custom_target(self, target):
             elem.add_item('pool', 'console')
         full_name = Path(target.subdir, target.name).as_posix()
         elem.add_item('COMMAND', cmd)
-        elem.add_item('description', f'Generating {full_name} with a custom command{cmd_type}')
+        elem.add_item('description', target.description.format(full_name)+cmd_type)
```

Comment:
```suggestion
        elem.add_item('description', target.description.format(full_name) + cmd_type)
```
we use spaces around operators